circleguard/widgets.py

```diff
@@ -37,7 +38,7 @@ def __init__(self, parent):
 
     def keyPressEvent(self, event):
         key = event.key()
-        if key == Qt.Key_Left or key == Qt.Key_Right:
+        if key in (Qt.Key_Left, Qt.Key_Right):
```

Comment:
Not a fan of the more terse `object in list` when list is only two objects. If it were 4 or more I would agree with this, but when we're only comparing against two it's better to be explicit and do both comparisons. Pretty certain both compile to the same bytecode anyway, though I could be wrong there.